app/views/home/home.html.erb

```diff
@@ -1,18 +1,123 @@
-<div class="container">
-
 <% cache('feature_home-intro', skip_digest: true) do %>
   <%= feature("home-intro") %>
```

Comment:
This now needs to be updated with a final version of https://gist.github.com/jywarren/ae51e24b4bcc2242ff2d8083838ce41a once published!